README.md

```diff
@@ -209,6 +209,16 @@ This works on Linux and MacOS.
 cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ...
 ```
 
+### ... with `meson`
+
+When creating th einitial build directory with `meson`,
+it will automatically generate a `compile_commands.jon`
+file inside of `builddir`.
```

Comment:
```suggestion
When creating the initial build directory with `meson`,
it will automatically generate a `compile_commands.json`
file inside of `builddir`.
```